scania-digital-design-system/tegel
# feat(banner): prefix change

**Describe pull-request**  

- Updated the prefix for the componet.

**Solving issue**  
Fixes: https://tegel.atlassian.net/browse/DTS-1625

**How to test**  
1. Go to storybook
2. Check in component
3. Check the styling and the API.
